code/modules/mining/minebot.dm

```diff
@@ -9,7 +9,6 @@
 	icon_state = "mining_drone"
 	icon_living = "mining_drone"
 	status_flags = CANSTUN|CANWEAKEN|CANPUSH
-	stop_automated_movement_when_pulled = 1
```

Comment:
this is very much needed. Apparently when you have /type1/type2/type3 and the following variable settings:
type1/var/myvar = 0 
type2/myvar = 1
type3/myvar = 0

Then the last one counts as a duplicate despite overriding the value in type2.